### PR TITLE
Update link for AI/ML job

### LINKED ISSUE #607: Ammed old job advert blog posts once they're no-longer active
Searching for RSE Sheffield vacancy currently (for my potentially biased google search) finds the [May 12th blog post](https://rse.shef.ac.uk/blog/2022-05-12-rrse/) for the job vacnancy, rather than the currently open vacancy. 

This page does not mention any application deadlines, simply linking to the job advert which has dissapeared as it is no longer active. 

We should probably ammend pages like this after the applications have closed so it is clear that it is no longer available.

### PR DESCRIPTION
**Pull request by bobturneruk**
Resolves #607

### CODE REVIEW

File: _posts/2022-05-12-rrse.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ We have a [job advert out for an RSE](https://www.jobs.ac.uk/job/CRL803/research
 5. **Do useful stuff with code.** We are most interested in your ability to do useful things with code and work with other people. We don’t tend to ask questions about the theory of computer science in interviews. The interest is in modern software technology, code you’ve written, how it was used and how it could be improved.
 6. **Reward and recognition.** All of the Sheffield RSE team are employed within the academic careers pathway, a new flexible career development structure which recognises a broad range of research outputs including software. The pathway provides a clear route for promotion up to the level of professor.
 
-Please [view the advert to apply](https://www.jobs.ac.uk/job/CPR556/research-software-engineer), and see contact details for enquiries about the job. General enquires about the team are welcome at [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk).
+Please [view the advert to apply](ttps://www.jobs.ac.uk/job/CRL803/research-software-engineer-rse-associate-rse-for-machine-learning/), and see contact details for enquiries about the job. General enquires about the team are welcome at [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk).
```
Comment (ptheywood) [inline _posts/2022-05-12-rrse.md]:
```suggestion
Please [view the advert to apply](https://www.jobs.ac.uk/job/CRL803/research-software-engineer-rse-associate-rse-for-machine-learning/), and see contact details for enquiries about the job. General enquires about the team are welcome at [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk).
```

**Review by ptheywood — CHANGES_REQUESTED** [review]
New link missing the h. 

Additionally, is the plan to just re-use the same blog post dated 2022-05-12 forever for job adverts? Rather than instead linking to a new / next blog post about hiring.

If I were looking for a job, and found a post from 2.5 months ago I'd probably assume that it wasn't still active.

For now it's an improvement though, so fine once fixed.

**Comment by bobturneruk** [comment]
Thanks @ptheywood - we do have a new post for the new job https://rse.shef.ac.uk/blog/2022-07-18-interview/ The updated links in the old post were intended to pick up anyone who landed on it while we had a position open. I get that landing on the old post might be confusing. Maybe this is more do to with Google's indexing than our site. Is there anything else you'd suggest?

**Comment by ptheywood** [comment]
There are ways to ask google not to index specific pages, e.g. `<meta name="robots" content="noindex">` but as they need to go in the `<head>` might be faffy to set up subject to the theme structure (and we don't control the webserver, so can't  make http header changes).

This won't be immediate (google must re-crawl the page) but should result in it being delisted from all search engines which pay attention to robots meta tags (i.e. nice ones). 

Otherwise we could kill the pages and 301 redirect them to the most up to date page / home page, depends how much we care about persisting history (i.e. if the page has more info or not).

**Review by ptheywood — APPROVED** [review]

**Comment by bobturneruk** [comment]
Thanks @ptheywood! I think we probably do need some better processes around this, especially if we're recruiting more regularly.

**Pull request merged**